kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -195,8 +209,48 @@ server {
         }
     }
 
+    location @error_400 {
+        default_type application/json;
+        return 400 '{"error_message": "Bad request", "error_code": "bad-request"}';
+    }
+
     location @error_401 {
         default_type application/json;
         return 401 '{"error_message": "Access denied.", "error_code": "unauthorized"}';
     }
+
+    location @error_403 {
+        default_type application/json;
+        return 403 '{"error_message": "Not permission for this page", "error_code": "forbidden"}';
```

Comment:
This part repeats in both nginx instances. Extract repeating parts into a snipped and just include them in both. We don't want repeated code that can easily go out of sync.